include/mbgl/util/geo.hpp

```diff
@@ -44,21 +44,39 @@ class ProjectedMeters {
 
 class LatLngBounds {
 public:
-    LatLng sw = {-90, -180};
-    LatLng ne = {90, 180};
-
-    LatLngBounds() {}
+    static LatLngBounds world() {
+        return LatLngBounds({-90, -180}, {90, 180});
+    }
 
-    LatLngBounds(const LatLng& sw_, const LatLng& ne_)
-        : sw(sw_), ne(ne_) {}
+    static LatLngBounds empty() {
+        LatLngBounds bounds = world();
+        std::swap(bounds.sw, bounds.ne);
+        return bounds;
+    }
 
-    static LatLngBounds getExtendable() {
-        LatLngBounds bounds;
-        return { bounds.ne, bounds.sw };
+    static LatLngBounds hull(const LatLng& a, const LatLng& b) {
```

Comment:
The naming of this method isn’t incredibly obvious. Could you add a documentation comment so its purpose is clear?